diff --git a/linssid-app/linssid-app.pro b/linssid-app/linssid-app.pro
index 13fd75e..111eb88 100644
--- a/linssid-app/linssid-app.pro
+++ b/linssid-app/linssid-app.pro
@@ -3,7 +3,7 @@ DESTDIR = dist/Release/GNU-Linux-x86
 TARGET = linssid
 VERSION = 3.6
 CONFIG -= debug_and_release app_bundle lib_bundle
-CONFIG += release 
+CONFIG += release
 PKGCONFIG +=
 QT = core gui widgets svg
 SOURCES += AboutBox.cpp Getter.cpp MainForm.cpp main.cpp prefsDialog.cpp
@@ -12,35 +12,36 @@ FORMS += AboutBox.ui MainForm.ui prefsDialog.ui
 RESOURCES +=
 TRANSLATIONS +=
 OBJECTS_DIR = build/Release/GNU-Linux-x86
-MOC_DIR = 
-RCC_DIR = 
-UI_DIR = 
+MOC_DIR =
+RCC_DIR =
+UI_DIR =
 QMAKE_CC = gcc
 QMAKE_CXX = g++
-DEFINES += 
-INCLUDEPATH += /usr/include/boost
-INCLUDEPATH += /usr/include/qt5
-INCLUDEPATH += /usr/include/qwt
+DEFINES +=
+#INCLUDEPATH += /usr/include/boost
+#INCLUDEPATH += /usr/include/qt5
+#INCLUDEPATH += /usr/include/qwt
 LIBS += -lboost_regex
-LIBS += /usr/lib/libqwt-qt5.so.6
+#LIBS += /usr/lib/libqwt-qt5.so.6
+LIBS += -lqwt
 QMAKE_CXXFLAGS += -std=c++11
 #
 TARGET = linssid
-target.path = /usr/sbin
+target.path = $$PREFIX/sbin
 INSTALLS += target
-targetexec.path = /usr/bin
+targetexec.path = $$PREFIX/bin
 targetexec.files=linssid-pkexec
 INSTALLS += targetexec
-dtop.path = /usr/share/applications
+dtop.path = $$PREFIX/share/applications
 dtop.files = linssid.desktop
 INSTALLS += dtop
-icons.path = /usr/share/pixmaps
+icons.path = $$PREFIX/share/pixmaps
 icons.files = *.png *.svg
 INSTALLS += icons
-data.path = /usr/share/linssid
+data.path = $$PREFIX/share/linssid
 data.files = oui.datc linssid-pause.png linssid-start.png
 INSTALLS += data
-policy.path = /usr/share/polkit-1/actions
+policy.path = $$PREFIX/share/polkit-1/actions
 policy.files = com.warsev.pkexec.linssid.policy
 INSTALLS += policy
 
